ensure_tiller_installed.go

```diff
@@ -207,10 +208,18 @@ func (c *Client) EnsureTillerInstalledWithValues(ctx context.Context, values []s
 
 			return nil
 		}
-		b := backoff.NewExponential(1*time.Minute, 5*time.Second)
+
+		// maxWait is configurable so we can use more time in e2e tests
+		// than in operators.
```

Comment:
The current value is 60 seconds which I set because we install tiller in e2e tests and in operators.

But 60 seconds is too long for cluster-operator so I'm making it configurable.